app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from datetime import date
+from typing import Union
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Union[str, Exception]:
```

Comment:
Do not indicate raised `Exception` in return type annotation, there is [no need for it](https://peps.python.org/pep-0484/#exceptions). Leave just `str`.